### PR TITLE
WebClub Landing Page

### PR DESCRIPTION
**Pull request by marcdhi**

### CODE REVIEW

File: src/app/layout.js [hunk 1]
```diff
@@ -6,14 +6,22 @@ import Footer, { MinimalSocialsFooter } from "@/components/footer";
 const inter = Inter({ subsets: ["latin"] });
```
Comment (ahmedfahim21) [inline src/app/layout.js:6]: If you dont want these font, remove this and its class. Declare everything from globals.css

File: src/app/page.js [hunk 2]
```diff
@@ -1,35 +1,52 @@
 "use client";
 import { BackgroundBeams, Logo } from "@/components/background-beams";
-
+import { motion } from "framer-motion";
+import { HeroHighlight, Highlight } from "@/components/ui/hero-highlight";
 import {
```
Comment (ahmedfahim21) [inline src/app/page.js:5]: 1 liner please

File: src/components/navbar.jsx [hunk 3]
```diff
@@ -7,19 +7,19 @@ const Navbar = () => {
   const toggleNavbar = () => {
     setIsOpen(!isOpen);
   };
-
+  // text-[#b3d3f6]
```
Comment (ahmedfahim21) [inline src/components/navbar.jsx:10]:
```suggestion
```

File: src/app/globals.css [hunk 4]
```diff
@@ -2,6 +2,14 @@
 @tailwind components;
 @tailwind utilities;
 
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  /* background-color: #0F0F15; */
```
Comment (ahmedfahim21) [inline src/app/globals.css:10]:
```suggestion
```

File: src/components/orbits-main.jsx
Comment (ahmedfahim21) [inline src/components/orbits-main.jsx:1]: Is this used somewhere? I am confused

File: src/components/what-we-do.jsx [hunk 5]
```diff
@@ -1,38 +1,133 @@
...
+
+
+
+export function BentoGrids() {
+  return (
+    <BentoGrid className="max-w-7xl mx-auto md:auto-rows-[20rem]">
+      {items.map((item, i) => (
+        <BentoGridItem
+          key={i}
+          title={item.title}
```
Comment (ahmedfahim21) [inline src/components/what-we-do.jsx:88]: A background picture(an illustration representing the what we do) with low opacity for each grid might look good

**Review by ahmedfahim21 — COMMENTED** [review]
Overall the alternate black and off white bg is looking good

**Comment by marcdhi** [comment]
Hero looks a bit plain imo, we'll have to change that. Or something similar to [bluelearn.in](bluelearn.in).

What do you think? @Shubham-Rasal, @ahmedfahim21, @j-hariharan

**Pull request merged**